simple_bank/test/simple_bank_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule SimpleBankTest do
+  use ExUnit.Case, async: false
+
+  alias SimpleBank.Account
+
+  setup do
+    {:ok, bank_pid} = SimpleBank.start_link([%Account{balance: 100, id: "test_id", name: "Test"}])
+    {:ok, bank: bank_pid}
+  end
+
+  describe "register/2" do
+    test "creates a new account and generates an account id", %{bank: bank_pid} do
+      {:ok, account_id} = SimpleBank.register(bank_pid, "Another Test Account")
+      assert is_binary(account_id)
+    end
+
+    test "raises an error for existing account names", %{bank: bank_pid}  do
+      {:error, :existing_account} = SimpleBank.register(bank_pid, "Test")
+    end
+  end
+
+  describe "deposit/3" do
+    test "increases the account balance by the deposited amount", %{bank: bank_pid}  do
+      assert {:ok, 110} == SimpleBank.deposit(bank_pid, "test_id", 10)
+    end
+
+    test "does not allow deposits of negative ammounts", %{bank: bank_pid}  do
```

Comment:
If I am not mistaken `ammount` should have a single `m`. This also occurs on lines 51 and 55.

Reply:
You're not mistaken.  I just have awful spelling 😞